### ISSUE TITLE
Dynamic Action Settings

### ISSUE THREAD

**Issue by pandamom8414** [s1m0nB/apex-plugin-apexbcreader#3]
Hi! I was wondering if you could explain the dynamic action settings to view the value of the barcode?

**Comment by rakibcse86** [comment]
please tell me how to set dynamic action? its emergency.....

**Comment by fwijnhoven** [comment]
did someone findout, I want to know this too !

**Comment by MohamedHeba** [comment]
Does anyone figure out how dynamic action work?

**Comment by step2live** [comment]
In need it too :(

**Comment by CInglez-STC** [comment]
Same here. Did anybody get it working? Is there any alternative?

**Comment by fwijnhoven** [comment]
We used it for for more then a year for an apex-appication on the mobile-phones, but we had problems with the newest android-smartphones with multiple cameras.
We switched to Dynasoft-Barcode-Reader, up till now our apex-applications on the mobile-phones are working without scan-problems, barcodes and qr-codes.
Regards,

**Comment by CInglez-STC** [comment]
fwijnhoven, thanks for the feedback. AFAIK, the Dynasoft solution works, but it is proprietary and not "free", right. This is not a deal breaker, but it makes things slower on my side. I don't know the values, I will inquiry them.

**Comment by fwijnhoven** [comment]
You are right, it’s not free, it is dependent of the number of scans you are doing in a year.
But software is working fine in apex.
